### PR TITLE
Created header component

### PR DESCRIPTION
**Pull request by magmat88**
Created header component with product name andnavigation links. Removed <View /> from App function.
Imported NavBar to header component, NavBar stayed in app.tsx.

### CODE REVIEW

File: src/app/app.tsx [hunk 1]
```diff
@@ -5,41 +5,71 @@ import { Link, Router, View } from 'react-navi';
 import { compose, mount, route, withView } from 'navi';
 import { Articles } from './components/articles/articles.component';
 import { Reports } from './components/reports/reports.component';
+import { HeaderComponent } from './components/header/header.component';
```
Comment (earendillo) [inline src/app/app.tsx:8]: rename to MainHeader; Component suffix is redundant here

File: src/app/components/articles/articles.component.tsx [hunk 2]
```diff
@@ -3,15 +3,14 @@ import { useStoreon } from 'storeon/react';
 import { Article } from '../../modules/articles/articles.state';
 import './articles.component.scss';
 
-
 export function Articles() {
     const { articles } = useStoreon('articles');
 
     return (
-        <div>
+        <div style={{ backgroundColor: 'pink' }}>
```
Comment (earendillo) [inline src/app/components/articles/articles.component.tsx:10]: get rid of inline styles

File: src/app/components/header/header.component.tsx [hunk 3]
```diff
@@ -1,9 +1,25 @@
+import { NavBar } from '../../app';
 import './header.component.scss';
 
 interface HeaderProps {
     appName: string;
 }
 
-export function Header({appName}: HeaderProps) {
-    return <div className='abc'>hello header</div>
+function Header({ appName }: HeaderProps) {
```
Comment (earendillo) [inline src/app/components/header/header.component.tsx:8]: no need to create separate component for wrapping an appName; moreover it has name of element from html5 specification

**Review by earendillo — CHANGES_REQUESTED** [review]

File: src/app/components/articles/articles.component.tsx (hunk 2 above)
Comment (magmat88) [reply src/app/components/articles/articles.component.tsx:10]: inline style removed

File: src/app/app.tsx (hunk 1 above)
Comment (magmat88) [reply src/app/app.tsx:8]: renamed

File: src/app/components/header/header.component.tsx (hunk 3 above)
Comment (magmat88) [reply src/app/components/header/header.component.tsx:8]: Header removed, appName prop moved into MainHeader

**Review by earendillo — APPROVED** [review]

**Pull request merged**